go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/operator-framework/operator-sdk v0.17.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.4.0 // indirect
```

Comment:
@DhritiShikhar this looks not correct, can you do a `go mod tidy`, no reason for this to be in the go.mod

Reply:
```
➜  build git:(master) git status
On branch master
Your branch is up to date with 'upstream/master'.

nothing to commit, working tree clean

➜  build git:(master) go mod tidy                                                     

➜  build git:(master) ✗ git status 
On branch master
Your branch is up to date with 'upstream/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   go.mod

no changes added to commit (use "git add" and/or "git commit -a")

➜  build git:(master) ✗ git diff  

➜  build git:(master) ✗ make local 
hack/crd.sh uninstall
clusterrole.rbac.authorization.k8s.io "build-operator" deleted
clusterrolebinding.rbac.authorization.k8s.io "build-operator" deleted
serviceaccount "build-operator" deleted
deployment.apps "build-operator" deleted
customresourcedefinition.apiextensions.k8s.io "buildstrategies.build.dev" deleted
customresourcedefinition.apiextensions.k8s.io "clusterbuildstrategies.build.dev" deleted
customresourcedefinition.apiextensions.k8s.io "builds.build.dev" deleted
customresourcedefinition.apiextensions.k8s.io "buildruns.build.dev" deleted
Error from server (NotFound): error when deleting "samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml": the server could not find the requested resource (delete clusterbuildstrategies.build.dev buildpacks-v3)
[WARN] Unable to delete resource 'samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml'
Error from server (NotFound): error when deleting "samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml": the server could not find the requested resource (delete clusterbuildstrategies.build.dev buildah)
[WARN] Unable to delete resource 'samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml'
Error from server (NotFound): error when deleting "samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml": the server could not find the requested resource (delete clusterbuildstrategies.build.dev kaniko)
[WARN] Unable to delete resource 'samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml'
Error from server (NotFound): error when deleting "samples/buildstrategy/source-to-image/buildstrategy_source-to-image_cr.yaml": the server could not find the requested resource (delete clusterbuildstrategies.build.dev source-to-image)
[WARN] Unable to delete resource 'samples/buildstrategy/source-to-image/buildstrategy_source-to-image_cr.yaml'
Error from server (NotFound): error when deleting "samples/buildstrategy/source-to-image/buildstrategy_source-to-image-redhat_cr.yaml": the server could not find the requested resource (delete clusterbuildstrategies.build.dev source-to-image-redhat)
[WARN] Unable to delete resource 'samples/buildstrategy/source-to-image/buildstrategy_source-to-image-redhat_cr.yaml'
clusterrole.rbac.authorization.k8s.io/build-operator created
clusterrolebinding.rbac.authorization.k8s.io/build-operator created
serviceaccount/build-operator created
deployment.apps/build-operator created
customresourcedefinition.apiextensions.k8s.io/buildstrategies.build.dev created
customresourcedefinition.apiextensions.k8s.io/clusterbuildstrategies.build.dev created
customresourcedefinition.apiextensions.k8s.io/builds.build.dev created
customresourcedefinition.apiextensions.k8s.io/buildruns.build.dev created
clusterbuildstrategy.build.dev/buildpacks-v3 created
clusterbuildstrategy.build.dev/buildah created
clusterbuildstrategy.build.dev/kaniko created
clusterbuildstrategy.build.dev/source-to-image created
clusterbuildstrategy.build.dev/source-to-image-redhat created
go mod vendor
go build -v -mod=vendor -o build/_output/bin/build-operator cmd/manager/main.go
github.com/golang/groupcache/lru
github.com/tektoncd/pipeline/pkg/apis/pipeline
github.com/redhat-developer/build/version
github.com/beorn7/perks/quantile
github.com/cespare/xxhash/v2
github.com/prometheus/common/internal/bitbucket.org/ww/goautoneg
sigs.k8s.io/controller-runtime/pkg/ratelimiter
github.com/tektoncd/pipeline/pkg/contexts
google.golang.org/protobuf/internal/impl
go.uber.org/zap/zapcore
k8s.io/apimachinery/pkg/util/mergepatch
github.com/PuerkitoBio/purell
k8s.io/apimachinery/third_party/forked/golang/json
github.com/prometheus/common/model
k8s.io/apimachinery/pkg/apis/meta/v1
github.com/emicklei/go-restful
github.com/go-openapi/swag
k8s.io/client-go/util/workqueue
github.com/prometheus/procfs/internal/fs
github.com/prometheus/procfs/internal/util
sigs.k8s.io/controller-runtime/pkg/healthz
github.com/prometheus/procfs
github.com/google/uuid
gomodules.xyz/jsonpatch/v2
github.com/go-openapi/jsonpointer
k8s.io/apimachinery/pkg/util/uuid
gopkg.in/fsnotify.v1
github.com/ghodss/yaml
go.uber.org/zap
github.com/go-openapi/jsonreference
knative.dev/pkg/kmp
github.com/go-openapi/spec
sigs.k8s.io/controller-runtime/pkg/webhook/internal/certwatcher
github.com/tektoncd/pipeline/pkg/list
github.com/tektoncd/pipeline/pkg/reconciler/pipeline/dag
github.com/mattbaird/jsonpatch
k8s.io/utils/pointer
github.com/Azure/go-autorest/autorest/date
github.com/Azure/go-autorest/tracing
github.com/dgrijalva/jwt-go
github.com/Azure/go-autorest/logger
cloud.google.com/go/compute/metadata
golang.org/x/oauth2/jws
github.com/Azure/go-autorest/autorest/adal
k8s.io/client-go/third_party/forked/golang/template
golang.org/x/oauth2/jwt
github.com/gophercloud/gophercloud
golang.org/x/oauth2/google
github.com/go-logr/zapr
k8s.io/client-go/util/jsonpath
sigs.k8s.io/controller-runtime/pkg/manager/signals
k8s.io/kube-openapi/pkg/common
google.golang.org/protobuf/internal/filetype
github.com/Azure/go-autorest/autorest
github.com/gophercloud/gophercloud/openstack/utils
github.com/gophercloud/gophercloud/openstack/identity/v3/tokens
github.com/gophercloud/gophercloud/pagination
google.golang.org/protobuf/runtime/protoimpl
k8s.io/apimachinery/pkg/runtime/serializer/protobuf
k8s.io/apimachinery/pkg/api/errors
k8s.io/apimachinery/pkg/apis/meta/v1/unstructured
github.com/golang/protobuf/proto
k8s.io/api/core/v1
google.golang.org/protobuf/types/known/anypb
google.golang.org/protobuf/types/known/durationpb
google.golang.org/protobuf/types/known/timestamppb
github.com/golang/protobuf/ptypes/any
k8s.io/api/admissionregistration/v1
github.com/golang/protobuf/ptypes/duration
k8s.io/api/admissionregistration/v1beta1
k8s.io/api/auditregistration/v1alpha1
github.com/golang/protobuf/ptypes/timestamp
k8s.io/api/authentication/v1
k8s.io/apimachinery/pkg/runtime/serializer/versioning
k8s.io/api/authentication/v1beta1
k8s.io/apimachinery/pkg/runtime/serializer
k8s.io/api/authorization/v1
k8s.io/api/authorization/v1beta1
github.com/golang/protobuf/ptypes
k8s.io/api/certificates/v1beta1
github.com/googleapis/gnostic/extensions
k8s.io/api/coordination/v1
k8s.io/api/coordination/v1beta1
github.com/googleapis/gnostic/compiler
k8s.io/api/flowcontrol/v1alpha1
k8s.io/api/rbac/v1
github.com/googleapis/gnostic/OpenAPIv2
k8s.io/api/rbac/v1alpha1
k8s.io/api/rbac/v1beta1
k8s.io/client-go/pkg/apis/clientauthentication
k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1
k8s.io/client-go/pkg/apis/clientauthentication/v1beta1
k8s.io/client-go/rest/watch
k8s.io/client-go/plugin/pkg/client/auth/exec
k8s.io/apimachinery/pkg/api/meta
k8s.io/apimachinery/pkg/apis/meta/v1beta1
k8s.io/client-go/rest
k8s.io/apimachinery/pkg/apis/meta/internalversion
k8s.io/client-go/util/retry
k8s.io/client-go/tools/clientcmd/api/latest
sigs.k8s.io/controller-runtime/pkg/scheme
sigs.k8s.io/controller-runtime/pkg/event
github.com/prometheus/client_model/go
github.com/matttproud/golang_protobuf_extensions/pbutil
sigs.k8s.io/controller-runtime/pkg/predicate
k8s.io/api/admission/v1beta1
k8s.io/client-go/tools/pager
k8s.io/kube-state-metrics/pkg/metrics_store
sigs.k8s.io/controller-runtime/pkg/log/zap
k8s.io/apimachinery/pkg/api/equality
k8s.io/kube-state-metrics/pkg/metric
github.com/prometheus/client_golang/prometheus/internal
k8s.io/client-go/dynamic
k8s.io/client-go/tools/cache
k8s.io/client-go/tools/auth
github.com/prometheus/common/expfmt
k8s.io/apimachinery/pkg/apis/meta/v1/validation
k8s.io/client-go/tools/clientcmd
github.com/operator-framework/operator-sdk/pkg/log/zap
github.com/Azure/go-autorest/autorest/azure
k8s.io/apimachinery/pkg/api/validation
k8s.io/client-go/plugin/pkg/client/auth/gcp
k8s.io/client-go/plugin/pkg/client/auth/oidc
github.com/gophercloud/gophercloud/openstack/identity/v2/tenants
github.com/prometheus/client_golang/prometheus
sigs.k8s.io/controller-runtime/pkg/client/config
github.com/gophercloud/gophercloud/openstack/identity/v2/tokens
k8s.io/client-go/plugin/pkg/client/auth/azure
github.com/gophercloud/gophercloud/openstack
k8s.io/client-go/listers/auditregistration/v1alpha1
k8s.io/client-go/listers/certificates/v1beta1
k8s.io/client-go/listers/admissionregistration/v1beta1
k8s.io/client-go/listers/admissionregistration/v1
k8s.io/client-go/listers/coordination/v1
k8s.io/client-go/listers/coordination/v1beta1
k8s.io/client-go/listers/flowcontrol/v1alpha1
k8s.io/client-go/listers/rbac/v1
k8s.io/client-go/listers/rbac/v1alpha1
k8s.io/client-go/listers/rbac/v1beta1
k8s.io/client-go/plugin/pkg/client/auth/openstack
k8s.io/client-go/plugin/pkg/client/auth
k8s.io/kube-openapi/pkg/util/proto
github.com/prometheus/client_golang/prometheus/promhttp
sigs.k8s.io/controller-runtime/pkg/metrics
k8s.io/apimachinery/pkg/util/strategicpatch
sigs.k8s.io/controller-runtime/pkg/internal/controller/metrics
sigs.k8s.io/controller-runtime/pkg/webhook/internal/metrics
k8s.io/api/batch/v1
k8s.io/api/discovery/v1alpha1
k8s.io/api/autoscaling/v1
k8s.io/api/autoscaling/v2beta1
k8s.io/api/autoscaling/v2beta2
k8s.io/api/apps/v1beta1
k8s.io/api/apps/v1
k8s.io/api/apps/v1beta2
k8s.io/api/discovery/v1beta1
k8s.io/api/batch/v1beta1
k8s.io/api/batch/v2alpha1
k8s.io/api/events/v1beta1
k8s.io/api/networking/v1
k8s.io/api/networking/v1beta1
k8s.io/api/node/v1alpha1
k8s.io/api/node/v1beta1
k8s.io/api/policy/v1beta1
k8s.io/api/scheduling/v1
k8s.io/api/extensions/v1beta1
k8s.io/api/scheduling/v1alpha1
k8s.io/api/scheduling/v1beta1
k8s.io/api/settings/v1alpha1
k8s.io/api/storage/v1
k8s.io/api/storage/v1alpha1
k8s.io/api/storage/v1beta1
k8s.io/client-go/tools/reference
github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1
github.com/redhat-developer/build/pkg/apis/build/v1alpha1
k8s.io/client-go/tools/record/util
k8s.io/client-go/tools/record
github.com/tektoncd/pipeline/pkg/apis/pipeline/pod
k8s.io/client-go/listers/apps/v1
sigs.k8s.io/controller-runtime/pkg/recorder
k8s.io/client-go/listers/apps/v1beta1
k8s.io/client-go/listers/apps/v1beta2
k8s.io/client-go/listers/autoscaling/v1
github.com/redhat-developer/build/pkg/apis
k8s.io/client-go/listers/autoscaling/v2beta1
github.com/coreos/prometheus-operator/pkg/client/versioned/scheme
k8s.io/client-go/listers/autoscaling/v2beta2
k8s.io/client-go/listers/batch/v1
k8s.io/client-go/listers/batch/v1beta1
github.com/coreos/prometheus-operator/pkg/client/versioned/typed/monitoring/v1
k8s.io/client-go/listers/batch/v2alpha1
k8s.io/client-go/listers/core/v1
k8s.io/client-go/listers/discovery/v1alpha1
k8s.io/client-go/listers/discovery/v1beta1
k8s.io/client-go/listers/events/v1beta1
k8s.io/client-go/listers/networking/v1
k8s.io/client-go/listers/networking/v1beta1
k8s.io/client-go/listers/node/v1alpha1
k8s.io/client-go/listers/node/v1beta1
k8s.io/client-go/listers/policy/v1beta1
k8s.io/client-go/listers/scheduling/v1
k8s.io/client-go/listers/scheduling/v1alpha1
k8s.io/client-go/listers/scheduling/v1beta1
k8s.io/client-go/listers/settings/v1alpha1
k8s.io/client-go/listers/storage/v1
k8s.io/client-go/listers/storage/v1alpha1
k8s.io/client-go/listers/storage/v1beta1
knative.dev/pkg/kmeta
knative.dev/pkg/apis
github.com/tektoncd/pipeline/pkg/apis/validate
github.com/tektoncd/pipeline/pkg/substitution
knative.dev/pkg/tracker
github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1
knative.dev/pkg/apis/duck
k8s.io/client-go/kubernetes/scheme
k8s.io/client-go/listers/extensions/v1beta1
k8s.io/client-go/discovery
k8s.io/client-go/kubernetes/typed/apps/v1
k8s.io/client-go/kubernetes/typed/admissionregistration/v1
k8s.io/client-go/kubernetes/typed/apps/v1beta1
k8s.io/client-go/kubernetes/typed/admissionregistration/v1beta1
k8s.io/client-go/kubernetes/typed/apps/v1beta2
k8s.io/client-go/kubernetes/typed/auditregistration/v1alpha1
k8s.io/client-go/kubernetes/typed/authentication/v1
k8s.io/client-go/kubernetes/typed/authentication/v1beta1
k8s.io/client-go/kubernetes/typed/authorization/v1
k8s.io/client-go/kubernetes/typed/authorization/v1beta1
k8s.io/client-go/restmapper
k8s.io/client-go/kubernetes/typed/autoscaling/v1
k8s.io/client-go/kubernetes/typed/autoscaling/v2beta1
k8s.io/client-go/kubernetes/typed/autoscaling/v2beta2
k8s.io/client-go/kubernetes/typed/batch/v1
k8s.io/client-go/kubernetes/typed/batch/v1beta1
k8s.io/client-go/kubernetes/typed/batch/v2alpha1
k8s.io/client-go/kubernetes/typed/certificates/v1beta1
k8s.io/client-go/kubernetes/typed/coordination/v1
k8s.io/client-go/kubernetes/typed/coordination/v1beta1
sigs.k8s.io/controller-runtime/pkg/client/apiutil
k8s.io/client-go/kubernetes/typed/core/v1
k8s.io/client-go/kubernetes/typed/discovery/v1alpha1
k8s.io/client-go/kubernetes/typed/discovery/v1beta1
k8s.io/client-go/kubernetes/typed/events/v1beta1
k8s.io/client-go/kubernetes/typed/extensions/v1beta1
k8s.io/client-go/kubernetes/typed/flowcontrol/v1alpha1
sigs.k8s.io/controller-runtime/pkg/client
k8s.io/client-go/kubernetes/typed/networking/v1
k8s.io/client-go/kubernetes/typed/networking/v1beta1
k8s.io/client-go/kubernetes/typed/node/v1alpha1
k8s.io/client-go/kubernetes/typed/node/v1beta1
k8s.io/client-go/kubernetes/typed/policy/v1beta1
k8s.io/client-go/kubernetes/typed/rbac/v1
k8s.io/client-go/kubernetes/typed/rbac/v1alpha1
k8s.io/client-go/kubernetes/typed/rbac/v1beta1
k8s.io/client-go/kubernetes/typed/scheduling/v1
k8s.io/client-go/kubernetes/typed/scheduling/v1alpha1
k8s.io/client-go/kubernetes/typed/scheduling/v1beta1
github.com/operator-framework/operator-sdk/pkg/k8sutil
k8s.io/client-go/kubernetes/typed/settings/v1alpha1
k8s.io/client-go/kubernetes/typed/storage/v1
k8s.io/client-go/kubernetes/typed/storage/v1alpha1
k8s.io/client-go/kubernetes/typed/storage/v1beta1
sigs.k8s.io/controller-runtime/pkg/cache/internal
k8s.io/client-go/tools/leaderelection/resourcelock
knative.dev/pkg/apis/duck/v1
sigs.k8s.io/controller-runtime/pkg/controller/controllerutil
sigs.k8s.io/controller-runtime/pkg/cache
k8s.io/client-go/kubernetes
github.com/operator-framework/operator-sdk/pkg/leader
github.com/operator-framework/operator-sdk/pkg/metrics
k8s.io/client-go/tools/leaderelection
sigs.k8s.io/controller-runtime/pkg/runtime/inject
sigs.k8s.io/controller-runtime/pkg/handler
sigs.k8s.io/controller-runtime/pkg/webhook/admission
knative.dev/pkg/apis/duck/v1beta1
sigs.k8s.io/controller-runtime/pkg/source/internal
sigs.k8s.io/controller-runtime/pkg/webhook
sigs.k8s.io/controller-runtime/pkg/source
sigs.k8s.io/controller-runtime/pkg/internal/controller
sigs.k8s.io/controller-runtime/pkg/internal/recorder
k8s.io/client-go/informers/internalinterfaces
sigs.k8s.io/controller-runtime/pkg/leaderelection
github.com/operator-framework/operator-sdk/pkg/kube-metrics
sigs.k8s.io/controller-runtime/pkg/manager
k8s.io/client-go/informers/admissionregistration/v1beta1
k8s.io/client-go/informers/auditregistration/v1alpha1
k8s.io/client-go/informers/apps/v1beta1
k8s.io/client-go/informers/apps/v1beta2
k8s.io/client-go/informers/apps/v1
k8s.io/client-go/informers/admissionregistration/v1
k8s.io/client-go/informers/autoscaling/v1
sigs.k8s.io/controller-runtime/pkg/controller
github.com/redhat-developer/build/pkg/controller/build
k8s.io/client-go/informers/autoscaling/v2beta1
k8s.io/client-go/informers/autoscaling/v2beta2
k8s.io/client-go/informers/batch/v1
k8s.io/client-go/informers/admissionregistration
k8s.io/client-go/informers/auditregistration
k8s.io/client-go/informers/batch/v1beta1
k8s.io/client-go/informers/batch/v2alpha1
k8s.io/client-go/informers/apps
k8s.io/client-go/informers/certificates/v1beta1
k8s.io/client-go/informers/coordination/v1
k8s.io/client-go/informers/coordination/v1beta1
k8s.io/client-go/informers/core/v1
k8s.io/client-go/informers/autoscaling
k8s.io/client-go/informers/batch
k8s.io/client-go/informers/discovery/v1alpha1
k8s.io/client-go/informers/discovery/v1beta1
k8s.io/client-go/informers/events/v1beta1
k8s.io/client-go/informers/extensions/v1beta1
k8s.io/client-go/informers/coordination
k8s.io/client-go/informers/certificates
k8s.io/client-go/informers/flowcontrol/v1alpha1
k8s.io/client-go/informers/networking/v1
k8s.io/client-go/informers/events
k8s.io/client-go/informers/core
k8s.io/client-go/informers/networking/v1beta1
k8s.io/client-go/informers/discovery
k8s.io/client-go/informers/node/v1alpha1
k8s.io/client-go/informers/extensions
k8s.io/client-go/informers/node/v1beta1
k8s.io/client-go/informers/policy/v1beta1
k8s.io/client-go/informers/flowcontrol
k8s.io/client-go/informers/rbac/v1
k8s.io/client-go/informers/networking
k8s.io/client-go/informers/rbac/v1alpha1
k8s.io/client-go/informers/rbac/v1beta1
k8s.io/client-go/informers/scheduling/v1
k8s.io/client-go/informers/node
k8s.io/client-go/informers/scheduling/v1alpha1
k8s.io/client-go/informers/policy
k8s.io/client-go/informers/scheduling/v1beta1
k8s.io/client-go/informers/settings/v1alpha1
k8s.io/client-go/informers/storage/v1
k8s.io/client-go/informers/storage/v1alpha1
k8s.io/client-go/informers/storage/v1beta1
k8s.io/client-go/informers/rbac
github.com/redhat-developer/build/pkg/controller/buildstrategy
github.com/redhat-developer/build/pkg/controller/clusterbuildstrategy
k8s.io/client-go/informers/scheduling
k8s.io/client-go/informers/settings
k8s.io/client-go/informers/storage
k8s.io/client-go/informers
knative.dev/pkg/configmap
github.com/tektoncd/pipeline/pkg/apis/config
github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1
github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1
github.com/redhat-developer/build/pkg/controller/buildrun
github.com/redhat-developer/build/pkg/controller
command-line-arguments
operator-sdk run --local --operator-flags="--zap-level=1 --zap-level=debug --zap-encoder=console"
INFO[0000] Running the operator locally in namespace default. 
2020-05-15T16:58:01.158+0530	INFO	cmd	Operator Version: 0.0.1
2020-05-15T16:58:01.158+0530	INFO	cmd	Go Version: go1.14.2
2020-05-15T16:58:01.158+0530	INFO	cmd	Go OS/Arch: linux/amd64
2020-05-15T16:58:01.158+0530	INFO	cmd	Version of operator-sdk: v0.17.0
2020-05-15T16:58:01.159+0530	INFO	leader	Trying to become the leader.
2020-05-15T16:58:01.159+0530	INFO	leader	Skipping leader election; not running in a cluster.
2020-05-15T16:58:03.366+0530	INFO	controller-runtime.metrics	metrics server is starting to listen	{"addr": "0.0.0.0:8383"}
2020-05-15T16:58:03.366+0530	INFO	cmd	Registering Components.
2020-05-15T16:58:03.366+0530	DEBUG	kubemetrics	Starting collecting operator types
2020-05-15T16:58:03.366+0530	DEBUG	kubemetrics	Generating metric families	{"apiVersion": "build.dev/v1alpha1", "kind": "Build"}
2020-05-15T16:58:05.571+0530	DEBUG	kubemetrics	Generating metric families	{"apiVersion": "build.dev/v1alpha1", "kind": "BuildStrategy"}
^C
➜  build git:(master) ✗ git status
On branch master
Your branch is up to date with 'upstream/master'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   go.mod
	deleted:    vendor/github.com/pmezard/go-difflib/LICENSE
	deleted:    vendor/github.com/pmezard/go-difflib/difflib/difflib.go
	deleted:    vendor/github.com/stretchr/testify/LICENSE
	deleted:    vendor/github.com/stretchr/testify/assert/assertion_format.go
	deleted:    vendor/github.com/stretchr/testify/assert/assertion_format.go.tmpl
	deleted:    vendor/github.com/stretchr/testify/assert/assertion_forward.go
	deleted:    vendor/github.com/stretchr/testify/assert/assertion_forward.go.tmpl
	deleted:    vendor/github.com/stretchr/testify/assert/assertion_order.go
	deleted:    vendor/github.com/stretchr/testify/assert/assertions.go
	deleted:    vendor/github.com/stretchr/testify/assert/doc.go
	deleted:    vendor/github.com/stretchr/testify/assert/errors.go
	deleted:    vendor/github.com/stretchr/testify/assert/forward_assertions.go
	deleted:    vendor/github.com/stretchr/testify/assert/http_assertions.go
	modified:   vendor/modules.txt

no changes added to commit (use "git add" and/or "git commit -a")

➜  build git:(master) ✗ 

```

Reply:
@qu1queee This is what I see after `go mod tidy`

Reply:
after your `go mod tidy` seems to me there is a change in go.mod, but when you do a `git diff` you do not see it. Can you verify this? `github.com/stretchr/testify v1.4.0` should be removed completely from go.mod.

Reply:
After `go mod tidy`, `github.com/stretchr/testify v1.4.0` is removed.

This is what I see -> https://youtu.be/ifhviEq19z4

Reply:
ok, but you still adding `github.com/stretchr/testify v1.4.0 // indirect` , can you ammend to your PR and remove it, so we can merge? btw nice video :)